x-pack/plugins/security_solution/public/detections/components/rules/step_rule_actions/get_schema.ts

```diff
@@ -21,7 +21,8 @@ export const getSchema = ({
   actions: {
     validations: [
       {
-        validator: validateRuleActionsField(actionTypeRegistry),
+        // Throttled validator is necessary here to prevent error validation flashing when first adding an action
+        validator: throttledValidateRuleActionsField(actionTypeRegistry),
```

Comment:
Lodash `throttle` potentially could've been used here instead, however [it doesn't work with async functions](https://github.com/lodash/lodash/issues/4815), so just deferred to leveraging actual cancellable async validation via the hooks form lib itself ([docs](https://docs.elastic.dev/form-lib/examples/validation#cancel-asynchronous-validation)).